answers/Properties/CorrectMultipleErrorsTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        public $address;
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
```

Comment:
Also this one.

Reply:
This similar to the given sample code in the q.md that we used to present to the learners. Please check this one. Thank you.

![image](https://user-images.githubusercontent.com/25945790/73618946-e375a280-4665-11ea-9af3-e642d641441b.png)